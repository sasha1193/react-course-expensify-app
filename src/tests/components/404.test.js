import React from 'react';
import { shallow } from 'enzyme';
import NotFound from '../../components/404';

test('should test 404 page', () => {
    const wrapper = shallow(<NotFound />);
    expect(wrapper).toMatchSnapshot();
})