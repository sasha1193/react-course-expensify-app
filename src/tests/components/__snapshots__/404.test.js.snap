// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should test 404 page 1`] = `
<div>
  404 - 
  <Link
    to="/"
  >
    Go HOME
  </Link>
</div>
`;
